ani-cli

```diff
@@ -464,7 +464,7 @@ scrape=Query
 debug=0
 choice=
 auto_play=0
-project_root="$(dirname "$0")"
+project_root="$(dirname "$0")/../share/ani-cli"
```

Comment:
> but technically isn't it still project root?

It isn't.
Lets consider the current default where ani-cli sits in `/usr/local/bin/ani-cli`:
$(dirname $0) will return `/usr/local/bin`
Now you append `/../share/ani-cli`
This implies we either split ani-cli into two directories, or if we start putting it into `usr/share` like good linux citizens we have more conflicts.
Also it definitely doesn't belong into `usr/local/share`, thats only populated by `man` on my system
So all in all this should be completely removed and you probably misunderstood it

Reply:
i've since changed this
```diff
467c467
< project_root="$(dirname "$0")"
---
> project_root="$(dirname "$0" | sed 's/....$//')/share/ani-cli"
```
also everything is in share/ani-cli. UI, ani-cli, players/, etc.
im just symlinking bin/ani-cli to share/ani-cli/ani-cli